### DISCUSSION TITLE
flex-grow or flex-1 with scroll element

### DISCUSSION THREAD

**Discussion by amjadbouhouch** [tailwindlabs/tailwindcss#5170] [Help]
I know this is maybe simple but I couldn't find the right implementation for it.  
I need to make the **nested** div element **scrollable**.  
Like in this example:  
[https://play.tailwindcss.com/gDSrcimRmz](url)  

Thanks.

**Comment by alexanderbluhm** [comment]
Hi @amjadbouhouch 
Your link is not working. Here is a minimal example for a nested div: https://play.tailwindcss.com/ADBhmLmND4
Key is `overflow` here.
